quadro.cpp

```diff
@@ -0,0 +1,82 @@
+#include <stdio.h>
+#include <stdbool.h>
+#include <math.h>
+
+struct coefs {
+    double a;
+    double b;
+    double c;
+};
+
+coefs quadro_input() {
+    coefs coefs;
+    coefs.a = 0;
+    coefs.b = 0;
+    coefs.c = 0;
+    printf("Enter coefficient a: ");
```

Comment:
repetitive parts of codes should be moved to a separate function. look: 16-20, 22-26, 28-32